content/developers/overview/managing-deploy-keys.md

```diff
@@ -83,6 +83,32 @@ See [our guide on Git automation with tokens][git-automation].
 7. Select **Allow write access** if you want this key to have write access to the repository. A deploy key with write access lets a deployment push to the repository.
 8. Click **Add key**.
 
+##### Using multiple repositories on one server
+
+If you use multiple repositories on one server, you will need to generate a dedicated key pair for each one. You can't reuse a deploy key for multiple repositories.
+
+In the server's SSH configuration file (usually `~/.ssh/config`), add an alias entry for each repository. For example:
+
+```bash
+Host github.com-repo-0
+        Hostname github.com
+        IdentityFile=/home/user/.ssh/repo-0_deploy_key
+
+Host github.com-repo-1
+        Hostname github.com
+        IdentityFile=/home/user/.ssh/repo-1_deploy_key
+```
+
+* `Host github.com-repo-0` - The repository's alias.
+* `Hostname github.com` - Configures the alias to use the {% data variables.product.product_name %} server.
```

Comment:
For the GHES version of this page, this renders as:

> Configures the alias to use the GitHub Enterprise Server server.

We'll probably need to condition this sentence for each version to it reads a little better.

Reply:
Good catch! Rephrased this line for wider compatibility.